### ISSUE TITLE
Screen glitch issue with iOS Screen Broadcast

### ISSUE THREAD

**Issue by bikee1235** [ant-media/Ant-Media-Server#6754]
Getting Screen glitch issue when iOS Screen Broadcast to Antmedia Streamer server, When we are streaming  camera using browser, we are not getting any glitches in stream 

### Environment

* Operating system and version: 17.0.12+7-Ubuntu-1ubuntu220.04, iOS version 13 and above 
* Java version: openjdk 17.0.12 2024-07-16 
* Ant Media Server version: Enterprise Edition 2.11.3 20240906_0602

* Browser name and version: Chrome Version 130.0.6723.92 (Official Build) (arm64)

### Steps to reproduce
1. Open Antmedia app and Tap Start Broadcast icon
2.  Then Start Broadcast
3. Go to Antmedia and in WebRTCAppEE App and try to play the stream

### Expected behavior
Stream of iOS should be smooth and clearly visible 

### Actual behavior
There is a vertical white lines  showing in middle of screen when  scroll is happening in color Pages

**Comment by muratugureminoglu** [comment]
<div>Agent Mohit Dubey linked Freshdesk ticket <a href="https://antmedia.freshdesk.com/a/tickets/135193">135193</a> for this issue.</div><br/>

**Comment by mekya** [comment]
Hi @bikee1235 

Thank you for creating the issue.
I've tested this scenario again on my device and unfortunately, I could not see the pixelation. 

Let's proceed this way.

Check this out -> [(iOS SDK Screen capture is pixelating when screen changes)](https://github.com/ant-media/Ant-Media-Server/issues/4382). The request is coming from our friend and your colleague Shibu and after the fix he confirms that it's working for him.  When I checked the ticket, I see that we talked about an SDK zip file. I mean we have not deliver the SDK as a zip file for maybe 2+ years.  We deliver it as a SPM package. Is there a chance that you use an older version?

I'm happy to jump on a call and debug it together. 

Cheers
Oz
